tests/test_misc.py

```diff
@@ -193,13 +193,19 @@ def test_change_hash_type(app, sqlalchemy_datastore):
     assert response.status_code == 302
 
 
-def test_md5():
```

Comment:
While the new API using the default settings is technically equivalent, just for a sanity check since a 3rd party library  is used now (`passlib`), maybe this should also try to expllicitly reproduce and assert equal results based on the defaults of the old API:

```python
import hashlib
def legacy_hash(msg):
    return hashlib.md5(encode_string(data)).hexdigest()

assert md5(u'hello') == hash_data(u'hello')
assert md5(b'hello') == hash_data(b'hello')
```